docs/Custom-Brimcap-Config.md

```diff
@@ -160,22 +159,28 @@ analyzers:
   - cmd: /usr/local/bin/suricata-wrapper.sh
 ```
 
-A Brimcap analyzer has the following characteristics:
+To be invoked successfully by Brimcap, an analyzer process should have the
+following characteristics:
+
+1. It expects pcap input to be streamed to it via standard input (stdin).
+2. It is assumed that if stdin is closed on a process launched by the analyzer,
+that process will exit.
```

Comment:
@mattnibs: The way you originally described this caveat to me:

> It is assumed that if stdin is closed on a process the process will exit.

In that wording, the way the word "process" is used twice seemed ambiguous to me, since there's the analyzer process and the ones it invokes. I've tried to clarify that in my version, but please confirm that it's still accurate in my version.